lib/routes/account.js

```diff
@@ -2459,7 +2459,8 @@ module.exports = (
             .then(
               function () {
                 // Notify the devices that the account has changed.
-                push.notifyPasswordReset(accountResetToken.uid, devicesToNotify)
+                var uid = Buffer(accountResetToken.uid, 'hex')
```

Comment:
> Error: The first param –uid– of notifyPasswordReset must be a Buffer!

OK, but the very next line calls `db.account(accountResetToken.uid)`, which is clearly expecting its argument to be a buffer:

  https://github.com/mozilla/fxa-auth-server/blob/master/lib/db.js#L376

So there's something fishy going on.  I wonder if there's a mock object somewhere that's providing it as a hex string during the tests, while a live database would return it as a buffer.